api/v1alpha1/system_types.go

```diff
@@ -577,6 +643,17 @@ func (spec *SystemSidekiqSpec) Default() {
 	spec.Resources = InitializeResourceRequirementsSpec(spec.Resources, systemDefaultSidekiqResources)
 	spec.LivenessProbe = InitializeProbeSpec(spec.LivenessProbe, systemDefaultSidekiqLivenessProbe)
 	spec.ReadinessProbe = InitializeProbeSpec(spec.ReadinessProbe, systemDefaultSidekiqReadinessProbe)
+	if spec.Config == nil {
+		spec.Config = &SidekiqConfig{}
+	}
+
+	if sidekiqType == "billing" {
+		spec.Config.Default(systemDefaultSidekiqConfigBilling)
+	} else if sidekiqType == "low" {
+		spec.Config.Default(systemDefaultSidekiqConfigLow)
+	} else {
+		spec.Config.Default(systemDefaultSidekiqConfigDefault)
+	}
```

Comment:
```suggestion
	if sidekiqType == Billing {
		spec.Config.Default(systemDefaultSidekiqConfigBilling)
	} else if sidekiqType == Low {
		spec.Config.Default(systemDefaultSidekiqConfigLow)
	} else {
		spec.Config.Default(systemDefaultSidekiqConfigDefault)
	}
```

We didn't change this when we added an enum, better this way than hardcoding the string.